winter/http/throttling.py

```diff
@@ -99,7 +96,8 @@ def create_throttle_classes(
 
         if getattr(throttling_annotation, 'rate', None) is not None:
             num_requests, duration = _parse_rate(throttling_annotation.rate)
-            throttling_ = Throttling(num_requests, duration, throttling_annotation.scope)
+            throttling_scope = throttling_annotation.scope or route.method.full_name
```

Comment:
you can add flag is_annotation_from_component and use it like so:

```
if throttling_annotation.scope is not None:
   scope = throttling_annotation.scope
elif is_annotation_from_component:
    scope = str(route.method.component)
else:
    scope = route.method.full_name
```

Reply:
Is this flag means that class throttling should be back?

Reply:
`is_annotation_from_component  = route.method.annotations.get_one_or_none(ThrottlingAnnotation) is None`

Reply:
```
        is_annotation_from_component = False
        throttling_annotation = route.method.annotations.get_one_or_none(ThrottlingAnnotation)
        if throttling_annotation is None:
            is_annotation_from_component = True
            throttling_annotation = base_throttling_annotation

        if getattr(throttling_annotation, 'rate', None) is not None:
            num_requests, duration = _parse_rate(throttling_annotation.rate)
            if throttling_annotation.scope is not None:
                scope = throttling_annotation.scope
            elif is_annotation_from_component:
                scope = str(route.method.component)
            else:
                scope = route.method.full_name
            throttling_ = Throttling(num_requests, duration, scope)
            throttling_by_http_method_[route.http_method.lower()] = throttling_
```

Reply:
It seems that 
`base_throttling_annotation = component.annotations.get_one_or_none(ThrottlingAnnotation)`
is no more needed since throttling for classes is moved out